src/object/authenticate.c

```diff
@@ -7205,6 +7146,8 @@ au_perform_login (const char *name, const char *password, bool ignore_dba_privil
 		{
 		  error = AU_SET_USER (user);
```

Comment:
no need to check error?

Reply:
Removed call sc_set_current_schema in next commit.